### ISSUE TITLE
Which lr_scheduler do you choose?

### ISSUE THREAD

**Issue by zzzzzzyang** [Sunner4nwpu/TEMMA#3]
Thanks for your work. I want to know which lr scheduler do you choose? You did not mention in your paper. Thanks a lot. :)

**Comment by Sunner4nwpu** [comment]
hi, the lr is fixed with 3e-4
